src/ephy-bookmark-properties-grid.c

```diff
@@ -328,6 +336,7 @@ ephy_bookmark_properties_grid_constructed (GObject *object)
   /* Set text for name entry */
   gtk_entry_set_text (GTK_ENTRY (self->name_entry),
                       ephy_bookmark_get_title (self->bookmark));
+  self->prev_name = g_strdup (gtk_entry_get_text (GTK_ENTRY (self->name_entry)));
```

Comment:
I'm confused, can you explain what this is for? How is there already text in `self->name_entry` if the `EphyBookmarksPropertiesGrid` is only now being constructed?